lib/Dialect/FIRRTL/Transforms/InferResets.cpp

```diff
@@ -767,6 +767,8 @@ void InferResetsPass::traceResets(CircuitOp circuit) {
                         op.getLoc());
           })
           .Case<Forceable>([&](Forceable op) {
+            if (auto node = dyn_cast<NodeOp>(op.getOperation()))
+              traceResets(node.getResult(), node.getInput(), node.getLoc());
```

Comment:
If this is mutually exclusive with the following, then `else if` would be more appropriate.

Reply:
Thanks! Nodes may be forceable, and if so we do also want to trace between the normal result and the rwprobe result, so I think this is correct as-is.